<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trigger_mode_dialog</class>
 <widget class="QDialog" name="trigger_mode_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>触发模式选择</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/logo.jpg</normaloff>../icon/logo.jpg</iconset>
  </property>
  <widget class="QPushButton" name="button_sure">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>200</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>触发模式</string>
   </property>
   <widget class="QRadioButton" name="radio_button_high">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>高于限值触发</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_low">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>低于限值触发</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
